x/datapool/genesis_test.go

```diff
@@ -69,6 +77,11 @@ func (suite genesisTestSuite) TestDataPoolInitGenesis() {
 	// check params
 	paramsFromKeeper := suite.DataPoolKeeper.GetParams(suite.Ctx)
 	suite.Require().Equal(params, paramsFromKeeper)
+
+	// check data pass redeem receipt
+	redeemReceiptFromKeeper, err := suite.DataPoolKeeper.GetDataPassRedeemReceipt(suite.Ctx, poolID, 1, redeemer)
+	suite.Require().NoError(err)
+	suite.Require().Equal(dataPassRedeemReceipt, redeemReceiptFromKeeper)
```

Comment:
Since this is the test for `InitGenesis()`, it would be better to check all the data pass redeem receipts are equal.
In other words, the `redeemReceiptFromKeeper` may be the result of `k.GetAllDataPassRedeemReceipts()`, and it should be equal to the `dataPassRedeemReceipts` above.

Reply:
Oh Thanks. I have a question. Is we need to apply for `pool` and `params` later?

Reply:
right. I think it would also be good to check all pools are equal.

Reply:
Thanks, I just leave these comments, not resolved.